controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/RoleDefinition.java

```diff
@@ -18,18 +18,17 @@
  */
 public enum RoleDefinition {
 
+    /** Base role which every user is part of. */
+    everyone(Policy.classifiedRead,
+            Policy.publicRead,
+            Policy.user,
+            Policy.tenantCreate),
+
     /** Deus ex machina. */
-    hostedOperator(Policy.operator),
+    hostedOperator(everyone, Policy.operator),
 
     /** Machina autem exspiravit. */
-    hostedSupporter(Policy.supporter,
-                    Policy.tenantCreatePublic),
-
-    /** Base role which every user is part of. */
-    everyone(Policy.classifiedRead,
-             Policy.publicRead,
-             Policy.user,
-             Policy.tenantCreate),
+    hostedSupporter(everyone, Policy.supporter),
```

Comment:
```suggestion
    hostedSupporter(Policy.supporter),
```
Anyone who is authenticated is automatically a member of the `everyone` role as well, but I guess that could be considered an implementation detail. 
I do know someone disliked thet role inheritance, though, and wanted to kill it.

Reply:
True, but that does not apply when the hostedSupporter role is used in unit tests. I'll look at the affected tests and consider changing it there instead.